packages/docs/src/pages/tutorial/hooks/use-host-element/problem/app.tsx

```diff
@@ -0,0 +1,11 @@
+import { component$, useHostElement, Host } from '@builder.io/qwik';
+
+export const App = component$(
+  () => {
+    const host = useHostElement();
+    return (
+      <Host onClick$={() => alert('Hello World!')}>Host element tag-name: {host.tagName}</Host>
```

Comment:
```suggestion
      <Host onClick$={() => alert('Hello World!')}>Host element tag-name: FILL IN THE HOST TAG NAME</Host>
```

Reply:
Thanks for the suggestion. I have incorporated it into the PR.